samples/unit-tests/series/datagrouping/demo.js

```diff
@@ -1016,4 +1016,35 @@ QUnit.test('The dataGrouping enabling/disabling.', function (assert) {
             the series.${prop} property should be deleted.`
         );
     });
+});
+
+QUnit.test('The dataGrouping with selected range.', function (assert) {
```

Comment:
Not really a big deal, but you can use ES6 arrow function here

Reply:
Not really a big deal again, but if there's only one argument, you don't need parentheses. But you can leave it as it is not :)